styles/style.css

```diff
@@ -123,29 +131,26 @@ label {
     margin: 0;
 }
 
-.top{
+.top {
     align-items: flex-start;
 }
 
-.bottom{
+.bottom {
     align-items: flex-end;
 }
 
 .play-card{
     animation-name: label-pull;
-    animation-duration: 100ms;
-    height: 75%;
+    animation-duration: 1000ms;
 }
 
-
 @keyframes label-pull {
```

Comment:
Nice use of keyframes. I know this was in the original code but I still think this is great.

Reply:
Dani helped us with the %'s, which was super helpful.  I'm pretty happy with how it worked out.